libjson-rpc-cpp.pc.cmake

```diff
@@ -0,0 +1,11 @@
+prefix=${FULL_PATH_INSTALL_PREFIX}
```

Comment:
Such files already exist for client, server, common and stubgernerator. Like here: https://github.com/cinemast/libjson-rpc-cpp/blob/master/cmake/libjsonrpccpp-client.pc.cmake

Maybe we should remove this. Because it does not take care of optional dependencies. For example one does not have to install both client and server. Than this pkg-config file would break the installation